### PR TITLE
Platform agnostic test commands

### PR DESCRIPTION
**Pull request by blagoev**

### CODE REVIEW

**Comment by blagoev** [comment]
retest this please

File: tests/test-runners/ava/package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "test": "ava"
   },
   "devDependencies": {
-    "ava": "^0.16.0"
+    "ava": "^0.16.0",
+    "realm": "../../.."
```
Comment (kristiandupont) [inline tests/test-runners/ava/package.json]: Why was this change necessary?

Comment (blagoev) [inline tests/test-runners/ava/package.json]: removed the need to do a symlink to the realm in the parent dir
